ui/app/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -132,6 +133,7 @@ export default class ImportWithSeedPhrase extends PureComponent {
       })
 
       setSeedPhraseBackedUp(true).then(() => {
+        completeOnboarding()
```

Comment:
This function is marked as `async` but I don't think there's any particular benefit to doing `await completeOnboarding()` here - I can update if it's semantically preferred.

Reply:
`await`-ing the call ensures any error is handled and shown to the user. Though admittedly this is not a great error handling pattern - such an error wouldn't exactly be relevant to users.

`await` would also give us an async stack trace if an error was thrown, ensuring we knew the `completeOnboarding` callsite. Async stack traces are supported by Chromium when using `Promise.all` or `async/await`, but not for `.then/.catch`.

I slightly prefer using `await` here, but it's not a big deal either way.

Reply:
Updated to use `await` 👍